### DISCUSSION TITLE
Style and content of new achievement badges trivialise actual achievements

### DISCUSSION THREAD

**Discussion by thclark** [community/community#28161] [Other Feature Feedback, Questions, & Ideas]
**It was REALLY AWESOME to be made an arctic code vault contributor.** Having that shield there under achievements on my profile was really a point of pride.

Recently, new achievements have been introduced, and I have a HUGE problem with both content and styling.

### Styling

Remember, people's GitHub profiles form their CVs - they need to convey professionalism.

Mine is linked to from LinkedIn, and lots of other places - it's the first place people go to if they need to assess whether they're any good, but it currently looks like Candy Crush Saga.

These weird cartoon emoji are massive, and distract the eye from actual content, and hugely trivialise my profile.

### Content

**Achievement badges should represent an achievement.** It should be possible to see someone's achievement badges and make some assessment of where they are as a contributor and developer.

Forced upon my profile I now have:
- "YOLO" which tells people that I merged a PR without a review, generally a bad practice but not really optional if you're the sole maintainer.
- "QUICKDRAW" which tells people that I opened an issue then closed it. On day 1 of using GitHub, probably.
- "PULL SHARK" which tells people that I made a pull request. Literally the most basic aspect of working with git.

I acknowledge the need to have achievement badges to encourage beginners, but many fragmented ones diminish those that are meaningful... none of the above are appropriate for a 15-year career developer who spends an average of 3 hours a day on GitHub. A simpler system might be appropriate - "Getting Started" could be replaced with "Moving up", then with "Git Hero" etc, and we'll know where people are at a glance without that panel getting so busy it's meaningless.

That would also be actually helpful, because in reviewing developer's CVs (i.e. their GitHub profiles!) it's nice to be able to see at a glance what level they're at with git, especially when hiring at a junior level.

There's a new STARSTRUCK one, which is fairly nice as it marks that some of your repos are getting traction in the OSS community, but the styling is still trivialising it, sadly.

### Suggested course of action

It would be good to keep achievements on my profile, rather than have to remove them entirely, because they have the potential to be (and indeed were) a great way of motivating myself and displaying credibility to the world.

For this not to be lost:

 - Make sure they're actual achievements, hard-won and meaningful and/or
 - Enable me to curate which ones are shown so my profile still looks professional and I can decide which are meaningful.

**Comment by heymage** [comment]
Hi @thclark, if you don't like the Achievements on your public profile, you can disable them in your settings at `Public Profile` > `Profile settings`.

Hope that helps

**Reply by thclark** [reply]
Hi @heymage thank you, I'm aware... but I had an **actual** achievement and want it to be there, as I was proud of it.

I've re-titled and edited my feedback above, partly to suppress the fact that I was pretty angry about this (which isn't helpful in feedback) and partly to clarify this aspect.

**Reply by heymage** [reply]
I see, maybe it would be possible to explicitly mark which achievements to show and which not on the public profile to gain power about that.

**Reply by thclark** [reply]
That would solve half the problem (i.e. enable us to restrict our own profile to useful information only).

The other half of the problem is the the noise affecting the usefulness GitHub, eg whilst browsing other people's profiles. When hiring devs, for example, the first thing to do is look at someone's GitHub profile to tell whether this is someone mature and established or whether they're raw and junior. 

Having meaningful achievements supports that, having a load of not-at-all-meaningful achievements prohibits it.

If you'd like some evidence that they're not very meaningful, you can check out [this other discussion](https://github.com/orgs/community/discussions/19146), full of people trying to get a YOLO badge by creating new repositories and fiddling with pull requests for no actual reason. Gamifying GitHub to incentivise good practice might not be a bad philosophy (above the scope of this discussion), but gamifying it to encourage bad practices and additional noise and clutter is surely a very bad plan.

There's the opportunity with Achievements to do something really credible and powerful for people's GitHub profiles, and I feel like this opportunity is being squandered.

**Reply by heymage** [reply]
I'm totally with you on this.

**Reply by aleebaba3594** [reply]
mark my reply i want get badge

**Comment by jfullstackdev** [comment]
yup, one valid request here is to select only the badges to be displayed, as of now, it's all or nothing

as for being not too professional looking or other says "cartoonish" and "childish", well, we need to balance all of the users of GitHub, others are actually very proud of it, as we will differ on our opinion and view

**Comment by whithajess** [comment]
I felt the exact same as @thclark was proud of my arctic code vault contributor badge and then felt patronised by the new badges - totally understand balancing all users of Github, this particular feature at the moment is not balanced.

I would suggest that if someone has made 100s or 1000s of PRs that QUICKDRAW or PULL SHARK should not show on their badges.

YOLO I am not even sure should exist but maybe thats personal opinion.

**Comment by thenithinbalaji** [comment]
Yeah true.
  
1. Achievements should be displayed in a separate tab and not under the profile. 
2. It should be fully transparent, and stuff like PRs and discussions made in private repos should not count for the process. 
3. If someone chose to turn achievements off, then, Github should display the "the user has turned off achievements" message since a few people already believe that more badges = more superior. 
4. Ability to report and remove someone's badge permanently if it's not authentic. Will prevent people from asking others to star their repo or mark theirs as answers in discussions. 
5. Except for Pull Shark and Starstruck badges, the other badges do not support or promote people to contribute to open-source in any way. They must be removed. 
6. Ability to turn off certain badges rather than turning off the entire set of achievements.

If you have any other suggestions, additions, or deletions, add them in the comments here https://github.com/community/community/discussions/28686#discussioncomment-3347349. Let's make Github hear!

**Comment by AyushJain001** [comment]
I was wondering the actual reason of achievement but may be it motivates to do more open source. Your information helped me figuring out the differences.

**Reply by aleebaba3594** [reply]
mark my reply i want get badge
